content/responses/folder_lock.yml

```diff
@@ -0,0 +1,61 @@
+---
+title: Folder Lock
+
+type: object
+
+x-box-resource-id: folder_lock
+x-box-tag: folder_locks
+
+description: |-
+  Folder locks define access restrictions placed by folder owners
+  to prevent specific folders from being moved or deleted.
+
+properties:
+  folder:
+    allOf:
+      - $ref: '#/components/schemas/Folder--Mini'
+      - description: The folder that the lock applies to.
+
+  id:
+    type: string
+    description: The unique identifier for this folder lock.
+    example: "12345678"
+
+  type:
+    type: string
+    description: The object type, always `folder_lock`.
+    example: "folder_lock"
+
+  created_by:
+    allOf:
+      - $ref: '#/components/schemas/User--Base'
+      - description: The user or group that created the lock.
```

Comment:
Is this a base user and not a mini user? Mini is our standard in APIs for embedding within other resources.

Reply:
Yep it's a base object - I had confirmed with the folder lock team and through independent tests against the cURL endpoints.